bookings/bookings.go

```diff
@@ -0,0 +1,44 @@
+// Wrapper for Zoho Bookings API from https://www.zohoapis.com/bookings/v1/json/
+
+package bookings
+
+import (
+	zoho "github.com/schmorrison/Zoho"
+	"math/rand"
+)
+
+const (
+	BookingsAPIEndpoint string = "https://www.zohoapis.in/bookings/v1/json/"
```

Comment:
View other subpackages to see how they handle the TLD.  Its very important because some users do not have access to the services behind some international TLDs.